actors/common/src/main/java/org/sunbird/metrics/actors/OrganisationMetricsBackgroundActor.java

```diff
@@ -126,7 +129,9 @@ private void orgCreationMetricsData(Request actorMessage) {
 
   @SuppressWarnings("unchecked")
   private void orgConsumptionMetricsData(Request actorMessage) {
-    ProjectLogger.log("In orgConsumptionMetricsExcel api");
+    ProjectLogger.log(
+        "OrganisationMetricsBackgroundActor:orgConsumptionMetricsData orgConsumptionMetricsExcel api",
```

Comment:
From:
"OrganisationMetricsBackgroundActor:orgConsumptionMetricsData orgConsumptionMetricsExcel api"

To:
"OrganisationMetricsBackgroundActor: orgConsumptionMetricsData called."